README.md

```diff
@@ -732,3 +732,47 @@ This can be done in two ways:
 **Note:** This only configures Numba to use the current RMM resource for allocations.
 It does not initialize nor change the current resource, e.g., enabling a memory pool.
 See [here](#memoryresource-objects) for more information on changing the current memory resource.
+
+### Using RMM with PyTorch
+
+[PyTorch](https://pytorch.org/docs/stable/notes/cuda.html) can use RMM
+for memory allocation.  For example, to configure PyTorch to use an
+RMM-managed pool:
+
+```python
+import rmm
+import torch
+
+rmm.reinitialize(pool_allocator=True)
+torch.cuda.memory.change_current_allocator(rmm.rmm_torch_allocator)
+```
+
+PyTorch and RMM will now share the same memory pool.
+
+You can, of course, use a custom memory resource with PyTorch as well:
+
+```python
+import rmm
+import torch
+
+# configure RMM to use a managed memory resource, wrapped with a
+# statistics resource adaptor that can report information about the
+# amount of memory allocated:
+mr = rmm.mr.StatisticsResourceAdaptor(rmm.mr.ManagedMemoryResource())
+rmm.mr.set_current_device_resource(mr)
+
+# configure PyTorch to use RMM for allocations:
+torch.cuda.change_current_allocator(rmm.rmm_torch_allocator)
```

Comment:
Both of the examples seemingly imply (at least to me, maybe that's just my bias) that the order of operations here is important. It might be worth mentioning that it's totally fine to call `change_current_allocator` and then change the resource, you'll still get the expected behavior (maybe just swap the order in one of these examples with a comment to that effect).

Reply:
I updated the docs - let me know what you think

Reply:
LGTM!